.github/release.yml

```diff
@@ -0,0 +1,24 @@
+# release.yml
+
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - octocat
+  categories:
```

Comment:
I'm thinking better categorizations 📝

Reply:
♻️ https://github.com/cookpad/terraform-aws-eks/pull/264/commits/01ec4f3c6cdb29b13b97a4ea7b4d3ea236137ea6

IMO, it's personal preference. If we need better categorizations, let's update the template.